### ISSUE TITLE
BatchNormalization's argument seems incorrect

### ISSUE THREAD

**Issue by Albelt** [yashkant/ENAS-Quantized-Neural-Networks#1]
Hi:
   Thanks for your work, here's my question
* In  pnas/model.py/parse_action(), action's input "x" is "channel_last", however the followed "BatchNormalization" function's argument axis is "1", I think the correct argument is "3" because the keras documentation writes as this:
```
 axis: Integer, the axis that should be normalized
            (typically the features axis).
            For instance, after a `Conv2D` layer with
            `data_format="channels_first"`,
            set `axis=1` in `BatchNormalization`.
```
the axis should be the last axis, isn't that? Or maybe my lib's version is different from yours?
 Here is my development environment:
```
linux 4.19.13
python3.6
tensorflow 1.8.0
keras 2.2.4
```

**Comment by yashkant** [comment]
Hi Albelt, 

You seem to have opened this issue in a wrong repository.

The concerned repository is: https://github.com/yashkant/PNAS-Binarized-Neural-Networks

Thanks for pointing this out. Yes, it is wrong, the axis=3 is the correct argument for it. 

If you wish to correct it for others as well, please send a PR [ here](https://github.com/yashkant/PNAS-Binarized-Neural-Networks/pulls) I'll be glad to merge it. :)
